test/unit/services/navigation_service_test.dart

```diff
@@ -36,16 +35,29 @@ void main() {
     });
 
     test('popHome calls NavigatorState.popUntil with /', () {
-      final navigatorState = MockNavigatorState();
-      final fakeNavKey = FakeGlobalKey(navigatorState);
+      final mockNavigatorState = MockNavigatorState();
+      final fakeNavKey = FakeGlobalKey(mockNavigatorState);
       final service = NavigationService(fakeNavKey);
 
+      /// The service calls [popUntil(ModalRoute.withName('/'))]
```

Comment:
the test creates the navigation service with a fake nav key (which in turn is created with a "mock navigator state") then calls popHome on the service and verifies the "mock navigator state" was used as expected. 

This is kind of a dumb test and will almost certainly go away when we migrate to Navigator2.0 but it took me frigging ages to get it to work so I can't make myself delete it just yet.